File: seeker/snippet/sum.go
//date: 2022-01-20T16:58:57Z
//url: https://api.github.com/gists/1662d644b19d0afc2e95aba89de585f5
//owner: https://api.github.com/users/m-murad

package math

func Sum(nums []int) int {
	var sum int
	for _, num := range nums {
		sum += num
	}
	return sum
}

File: seeker/snippet/sum.py
#date: 2025-03-11T16:53:51Z
#url: https://api.github.com/gists/00f38c59e5c37dd4bb2c3292a7ba84cb
#owner: https://api.github.com/users/mockqv

def sum(list):
    if not list:
        return 0
    return list[0] + sum(list[1:])